tee-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -178,6 +178,9 @@ pub enum TrustedCall {
 	handle_vcmp_error(Identity, Option<Identity>, VCMPError, H256),
 	#[codec(index = 17)]
 	send_erroneous_parentchain_call(Identity),
+	#[cfg(not(feature = "production"))]
```

Comment:
IMO It's OK that we keep it in enum even for prod - otherwise we'll need to have two different client-api packages.

Reply:
Since we explicitly set indices we can have this variant only for non prod mode while having single client-api package. If the client try to use it on prod mode the sidechain will fail on deconding. imho it would be ok.

Reply:
`tee-worker/client-api/parachain-api/prepare-build/interfaces/trusted_operations/definitions.ts`
We should add this varaint to `TrustedCall` definition in this file ^ as `_Unused_*`, `link_identity_callback`, `request_vc_callback`,  `handle_imp_error`, `handle_vcmp_error`, `send_erroneous_parentchain_call` should be added too - otherwise we will have problems if we add new variant with index 19 that should be used by `client-api`.

Reply:
Either way would work - I'd argue having it not guarded with prod feature has better maintainability and is less error-prone, while having it guarded makes sure we won't have such trusted call in prod whatsoever.

So your choice :) @Traf333